ember_debug/adapters/basic.js

```diff
@@ -1,6 +1,6 @@
 /* globals requireModule */
 /* eslint no-console: 0 */
-const Ember = window.Ember;
+const Ember = require('ember') || window.Ember;
```

Comment:
I think we need something like:

```suggestion
require.has('ember') ? require('ember') : window.Ember;
```

Reply:
Unfortunately this causes errors